deployment/morph/default.nix

```diff
@@ -14,5 +14,7 @@ in
   "${machines.marloweDashB.dns}" = configurations.pab "marlowe-dash-b";
   "${machines.webghcA.dns}" = configurations.webGhc "web-ghc-a";
   "${machines.webghcB.dns}" = configurations.webGhc "web-ghc-b";
+  "${machines.playgroundsA.dns}" = configurations.playgrounds "playgrounds-a";
```

Comment:
More of a detail really but, iirc the `A`s and `B`s here refer to the different deployments to the 2 different zones. Maybe you could just add some comment about that right here? Unless there is a different better place to point that out *or* it already _is_ mentioned somewhere and i am just simply not aware of it.

Thanks.